service/src/main/java/bio/terra/catalog/config/BaseDatabaseConfiguration.java

```diff
@@ -8,6 +8,7 @@
 import org.apache.commons.dbcp2.PoolingDataSource;
 import org.apache.commons.pool2.ObjectPool;
 import org.apache.commons.pool2.impl.GenericObjectPool;
+import org.springframework.web.bind.annotation.RestController;
```

Comment:
unused import?

Reply:
Got rid of it in most recent version. Probably forgot to list the file when updating branch.